docs/source/contributing.rst

```diff
@@ -37,8 +37,8 @@ To install ``soundata`` for development purposes:
     .. code-block:: console
 
         pip install .
-        pip install .[tests]
-        pip install .[docs]
+        pip install ."[tests]"
```

Comment:
Ok! btw, sorry I reviewed with an old account on PR #114. I noticed that here, apart from this change, the rest of the changes are also applied in #114. I guess that's not a problem though. You will first merge this one?

Reply:
Hi @genisplaja, that should not be a problem, I have to make changes in that so I'll update that and merge this one first.